index.js

```diff
@@ -148,6 +148,28 @@ const hasDevDependency = (dependency, packageJson) => {
   )
 }
 
+const runSValues = [/run-s/]
```

Comment:
`npm-run-all -s`
`npm-run-all --sequential`
`npm-run-all --serial`

should also detected